.prettierrc

```diff
@@ -21,6 +21,14 @@
             "options": {
                 "tabWidth": 2
             }
+        },
+        {
+            "files": "*.sql",
+            "options": {
+                "formatter": "sql-formatter",
+                "language": "tsql",
+                "uppercase": true
+            }
```

Comment:
same change needs to made to the .prettierrc in /boilerplate/files/.prettierrc

Reply:
updated